ruby_04-apis_and_scalability/js_refactor_tractor.md

```diff
@@ -325,6 +325,69 @@ __Discussion Points__
 
 #### Single Responsibility Principle && Code that Does Too Much
 
+A class or module should only have one reason to change.
+
+Anything that gives a class a reason to change should be considered a responsibility.
```

Comment:
Aaron. Have you considered working for Turing? Have you have coffee with Jorge yet?